src/new-components/text-field/text-field.module.css

```diff
@@ -53,3 +53,9 @@
 .inputWrapper:not(.bordered) input {
     padding: var(--tmp-vertical-padding) 10px;
 }
+
+.startIcon {
+    margin-right: calc(var(--reactist-spacing-xsmall) * -1);
+    padding-left: 10px;
+    cursor: text;
+}
```

Comment:
💭 Do you think `.startIcon` should have a max height? On buttons no matter how big the icon is it won't cause the button to be larger than normal, however that is not currently the case with text fields.